packages/augur-ui/src/modules/auth/helpers/login-account.ts

```diff
@@ -115,9 +115,9 @@ export const getCoreStats = (isLogged, loginAccount) => {
 };
 
 export const getEthReserve = () => {
-  const { loginAccount: { balances } } = AppStatus.get();
-  const ethNonSafeBN = createBigNumber(balances.ethNonSafe);
-  let desiredSignerEthBalance = createBigNumber(formatAttoEth(Number(DESIRED_SIGNER_ETH_BALANCE)).value);
+  const { env, loginAccount: { balances } } = AppStatus.get();
```

Comment:
destruct this so it's `{ env: { gsn: { desiredSignerBalanceInETH: gsnETHBalance } }, loginAccount...`

then later:
`let desiredSignerEthBalance = createBigNumber(formatAttoEth(Number(gsnETHBalance * 10**18)).value);` -- been trying to favor destructuring instead of `foo.bar.biz`